src/components/DataDecorator/constants.js

```diff
@@ -1,36 +0,0 @@
-/**
```

Comment:
I'd be interested in understanding if this will introduce a breaking change for any consumers leveraging these icons

Reply:
@SimonFinney that's a good point -- I'll return the file for now and just make sure it imports icons from `Decorator/constants` (not redefines them).